docs/source/description.rst

```diff
@@ -2,3 +2,17 @@ Alot is a terminal-based mail user agent for the notmuch mail system.
 It features a modular and command prompt driven interface
 to provide a full MUA experience as an alternative to the Emacs mode shipped
 with notmuch.
+What you can do whit alot:
+
+	* Read mail/thread
+	* Search mail
+	* Tag/Untag mail
+	* Filter mail	
+	* Save attachment
```

Comment:
probably should be `Save attachments`

Reply:
why attachments would be plural and mail would be singular ?